index.html

```diff
@@ -0,0 +1,126 @@
+                    <li><a href="#join" class="nav-list">Join</a></li>
+                    <li><a href="#sponsor" class="nav-list">Sponsor</a></li>
+                    <li><a href="#news" class="nav-list">News</a></li>
+                    <li><a href="#campain" class="campain">CC Campain</a></li>
+                </ul>
+            </div>
+            <div class="home-page">
+                <div class="title">
+                    <p class="greeting"><strong>Hello World!</strong></p>
+                    <h1>Artifitial Intelligence Webinar 2022</h1>
+                </div>
+                <div class="disc">
+                    <p>The life we are living today and how is the Artificial intelligence is evolving to our daily life
+                    </p>
+                </div>
+                <div class="time">
+                    <h4>2022.10.15(Mon.)~ 16(Tue.)</h4>
+                    <p>@Rwanda Convertion center of Kigali</p>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <section class="activities">
+        <h3> Main Program</h3>
+        <div class="allparts">
+            <div class="part">
+                <img src="./images/topic.png" alt="Menu" />
+                <p class="type">Topics</p>
+                <p class="explanation">Take look on the topics and ideas will be sharing in the Artificial Intelligence
+                    webinar</p>
+            </div>
+
+            <div class="part">
+                <img src="./images/connecticon.png" alt="Menu" />
+                <p class="type">Connect</p>
+                <p class="explanation"> came connect with world greatest AI Specialist who are from all part of the
+                    world</p>
+            </div>
+
+            <div class="part">
+                <img src="./images/forum.png" alt="Menu" />
+                <p class="type">Forum</p>
+                <p class="explanation">Have time to share your insight, opinion and ideas with presenters to the topic
+                </p>
+            </div>
+
+            <div class="part">
+                <img src="./images/collaboration.png" alt="Menu" />
+                <p class="type">collaborate</p>
+                <p class="explanation">Have time to share your insight, opinion and ideas with presenters to the topic
+                </p>
+            </div>
+            <div class="part">
+                <img src="./images/calendar.png" alt="Menu" />
+                <p class="type">Date</p>
+                <p class="explanation">Take a look on the important dates and the time in which will be discuss for the
+                    cartain topic </p>
+            </div>
+        </div>
+        <button class="clickme" type="button">Join AI Webinar 2022</button>
+        <a href="#" class="program">SEE THE WHOLE PROGRAM</a> 
+    </section>
+
+    <section class="speakers" id="speakers"> <h2>,</h2></section>
+
```

Comment:
The features should look almost identical to the original design, outside the content differences. 

- [ ] There fore I suggest adding a checked board on the speakers image as seen in the original design

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/159138584-3fdd19b9-09ab-4994-9c05-016277e8af7a.png) | ![image](https://user-images.githubusercontent.com/4247740/159138615-37ceb00f-0e78-464c-8f44-26ceee70971a.png) |

- [ ] As seen in the original screenshot, the speakers are arranged side by side. Kindly implement it.

Reply:
partially resolved